.github/workflows/ci.yml

```diff
@@ -16,9 +16,11 @@ jobs:
         with:
           node-version: "16"
           cache: "yarn"
+      - run: |
+          echo "${{ secrets.TEST_ENV }}" > .env
```

Comment:
Does this need to be a secret? If not, may be simpler to just hardcode it here in the workflow?

(I don't know what this is for, so feel free to ignore this comment)

Reply:
Hmm yeah, probably not now that I think about it.